lua/cmp/types/cmp.lua

```diff
@@ -74,6 +74,7 @@ cmp.ItemField.Menu = 'menu'
 ---@field public enabled fun():boolean|boolean
 ---@field public preselect cmp.PreselectMode
 ---@field public completion cmp.CompletionConfig
+---@field public window cmp.WindowConfig|nil
 ---@field public documentation cmp.DocumentationConfig|"false"
```

Comment:
We should remove documentation setting. I think.

Reply:
As in `window.documentation`? This particular `@class` still exists, just moved to `window`

Reply:
No. I mean we should remove `cmp.setup { documentation = { ... } }` because we will add the `window.documentation`.

Reply:
That was My thought as well. I did that in 36c8ba7c73b9be852ef34bd6f573c2b0dcd72aa0